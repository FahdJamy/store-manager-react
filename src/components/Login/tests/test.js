import React from 'react';
import { shallow } from 'enzyme'

import LoginForm from '../Login';

describe('tests LoginForm', ()=> {
    it ('it should render without rashing', () => {
        let wrapper = shallow(<LoginForm />)
        expect(wrapper).toMatchSnapshot()
    })
})
