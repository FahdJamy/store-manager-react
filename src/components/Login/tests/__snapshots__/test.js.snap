// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests LoginForm it should render without rashing 1`] = `ShallowWrapper {}`;
